src/protocol/mod.rs

```diff
@@ -108,6 +108,11 @@ impl<Stream> WebSocket<Stream> {
     pub fn get_mut(&mut self) -> &mut Stream {
         self.socket.get_mut()
     }
+
+    /// Change the configuration.
+    pub fn set_config(&mut self, set_func: impl FnOnce(&mut WebSocketConfig)) {
+        set_func(&mut self.config)
```

Comment:
Hm, but it could be misleading as the one may thing that modifying the configuration will immediately affect the websocket connection, which in fact would not happen (the internal `SocketFrame` is not modified).

I think that the configuration for a single websocket connection once on initialization should be sufficient, isn't it?

Reply:
`FrameSocket` does not have any internal configuration state, all parameters are passed at function calls. This `self.config` is the only place where configuration is stored and applied. Thus it takes effect immediately.